### PR TITLE
okok

### PR DESCRIPTION
**Pull request by ghost**
- [DEMO LINK](https://junglq.github.io/layout_snake/)
- [TEST REPORT LINK](https://junglq.github.io/layout_snake/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+.wrapper__block {
+  height: 300px;
+  width: 310px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 100px;
+  color: #fff;
+  flex-basis: 33.33%;
```
Comment (AlinaMatuschak) [inline src/style.css:19]: Can use calc

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]
Good start 👌. But check small details

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.wrapper__block {
+  min-height: 300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 100px;
+  color: #fff;
+  flex-basis: 33.33%;
```
Comment (AlinaMatuschak) [inline src/style.css:21]: Is the idea with calc not interesting?

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.wrapper__block:nth-child(5) {
+  background-color: rgb(20%, 0%, 0%);
+  order: 5;
+}
+
+.wrapper__block:nth-child(6) {
+  background-color: rgb(0%, 0%, 0%);
+  order: 4;
```
Comment (AlinaMatuschak) [inline src/style.css:49]: you can star from 1

**Review by AlinaMatuschak — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (junglq) [reply src/style.css:49]: dont understand

File: src/style.css (hunk 2 above)
Comment (AlinaMatuschak) [reply src/style.css:21]: Why?

File: src/style.css (hunk 3 above)
Comment (AlinaMatuschak) [reply src/style.css:49]: You don't have 1, 2, 3 orders. It's not a mistake, but try not to use too big values )

File: src/style.css (hunk 2 above)
Comment (AlinaMatuschak) [reply src/style.css:21]: It helps to read your code easily. Try to avoid fractional values

**Pull request closed without merging**